specification/components/schemas/extensions/ServiceSummaryViewExtension.yaml

```diff
@@ -0,0 +1,473 @@
+                        enum: 
+                          - MALE
+                          - FEMALE
+                          - MALE_AND_FEMALE
+                        example: 'MALE_AND_FEMALE'
+        - title: Additional support services provided by the Organisation where the Service is located at
+          type: object
+          required:
+            - url
+            - valueCodeableConcept
+          properties:
+            url:
+              type: string
+              enum: 
+                - additionalRequirementSupported
+              example: 'additionalRequirementSupported'
+            valueCodeableConcept:
+              type: object
+              required:
+                - coding
+              properties:
+                coding:
+                  type: array
+                  minItems: 1
+                  maxItems: 1
+                  items:
+                    type: object
+                    required:
+                     - system
+                     - code 
+                    properties:
+                      system:
+                        type: string
+                        enum:
+                          - https://fhir.nhs.uk/STU3/CodeSystem/eRS-AdditionalRequirementType-1
+                        example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-AdditionalRequirementType-1'
+                      code:
+                        type: string
+                        enum: 
+                          - TRANSPORT
+                          - INTERPRETER
+                          - ADVOCACY
+                        example: 'ADVOCACY'
+        - title: Indicates whether the Service requires a referral letter before they consider a referral to be valid
+          type: object
+          required:
+            - url
+            - valueBoolean
+          properties:
+            url:
+              type: string
+              enum:
+                - referralLetterRequired
+              example: 'referralLetterRequired'
+            valueBoolean:
+              type: boolean
+              example: true
+            valueReference:
+              type: object
+              required:
+                - identifier
+                - display
+              properties:
+                identifier:
+                  type: object
+                  required:
+                    - system
+                    - value
+                  properties:
+                    system:
+                      type: string
+                      enum:
+                        - https://directory.spineservices.nhs.uk/STU3/Organization
+                      example: 'https://directory.spineservices.nhs.uk/STU3/Organization'
+                    value:
+                      type: string
+                      example: 'R69'
+                display:
+                  type: string
+                  example: 'Cicero-Clinic'
+        - title: The specialty supported by the Service
```

Comment:
Perhaps this description could include a reference to the get reference data endpoint indicating that by calling this endpoint you can retrieve the full set of possible specialty values.

Reply:
Related APIS will be included in each endpoint's overview section.